javascript/modules/client/src/internal/encoding/plugins.ts

```diff
@@ -58,7 +58,7 @@ function unwrapErc20InitParams(params: IErc20PluginInstall): [string, string, Bi
   ]
 }
 
-export function encodeActionSetPluginConfig(params: IProposalSettings): Uint8Array {
+export function encodeActionSetPluginConfig(params: IPluginSettings): Uint8Array {
```

Comment:
Better like this?

```suggestion
export function encodeSetPluginSettingsAction(params: IPluginSettings): Uint8Array {
```

Reply:
I would say `encodeUpdatePluginSettingsAction` is a better name since the function that calls this function is `_buildUpdatePluginSettingsAction`